src/client/AccountsClient.js

```diff
@@ -10,13 +10,13 @@ import AccountsCommon from '../common/AccountsCommon';
 import { loggingIn } from './module';
 import ui from './ui';
 import type { AccountsOptionsType } from '../common/AccountsCommon';
+import type UserObjectType from '../common/UserObjectType';
 
-export type UserObjectType = {username: ?string, email: ?string, id: ?string};
 const isValidUserObject = (user: UserObjectType) => has(user, 'user') || has(user, 'email') || has(user, 'id');
 
 export type UserCreationInputType = {
   username: ?string,
-  password: ?string,
+  password: string,
```

Comment:
I think this is supposed to be nullable because of oauth or strategies where password is not required like phone number validation (Send a text to your phone number and realise this device is to be authenticated)
Another option would be to have these kind of strategies handled by different methods.

LMK what you think.